translations/monero-core_ro.ts

```diff
@@ -1079,235 +1079,242 @@ The following information will be deleted
 
 The old wallet cache file will be renamed and can be restored later.
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Ești sigur că vrei să reconstruiești cache-ul portofelului?
+Următoarele informații vor fi șterse
+- Adresele destinatarilor
+- Cheile Tx
+- Descrierile Tx
+
+Vechiul fișier cache va fi redenumit și va putea fi restaurat mai tâziu.
+</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="188"/>
         <source>Cancel</source>
-        <translation type="unfinished">Renunță</translation>
+        <translation>Renunță</translation>
     </message>
     <message>
         <location filename="../pages/settings/SettingsInfo.qml" line="202"/>
         <source>Invalid restore height specified. Must be a number.</source>
-        <translation type="unfinished"></translation>
+        <translation>Înălțimea de restaurare săecificată nu este validă. Trebuie să fie un număr.</translation>
```

Comment:
Should be 
`        <translation>Înălțimea de restaurare specificată nu este validă. Trebuie să fie un număr.</translation>
`